programs/drift/src/state/perp_market.rs

```diff
@@ -1212,6 +1212,8 @@ impl AMM {
                 msg!("Can't get oracle twap for quote asset");
                 Err(ErrorCode::DefaultError)
             }
+            // todo is this correct?
+            OracleSource::Prelaunch => Ok(Some(get_prelaunch_price(price_oracle)?.price)),
```

Comment:
since theres no track of twap on price_oracle you could just look at market account's `oracle_twap`?